tests/h/views/api_groups_test.py

```diff
@@ -138,10 +138,28 @@ def test_it_passes_request_params_to_group_create_service(self,  # noqa: N803
                                                               pyramid_request,
                                                               CreateGroupAPISchema,
                                                               group_service):
-        CreateGroupAPISchema.return_value.validate.return_value = {'name': 'My Group'}
+        CreateGroupAPISchema.return_value.validate.return_value = {
+          'name': 'My Group',
+          'description': 'How about that?',
+         }
         views.create(pyramid_request)
 
-        group_service.create_private_group.assert_called_once_with('My Group', pyramid_request.user.userid)
+        group_service.create_private_group.assert_called_once_with('My Group',
+                                                                   pyramid_request.user.userid,
+                                                                   description='How about that?')
+
+    def test_it_sets_description_to_none_if_not_present(self,  # noqa: N803
```

Comment:
`N803` is ignored globally in our linter config. It looks like you may be using flake8 but somehow without it reading our linter config?

Reply:
The problem is my editor, TBH. It doesn't seem possible to configure it in any way wherein it can handle multiple configs per project. So it picks up the top-level config OK, but ignores the config at the test level if I have the whole project open.

I think a workaround I can use is to open a separate editor session at the test level when I'm editing tests. Kind of kludgy, but I'll give it a whirl.